eng/scripts/Sdk-Generate.ps1

```diff
@@ -0,0 +1,144 @@
+param(
```

Comment:
I highly recommend writing scripts as advanced cmdlets, which use the `[CmdletBinding()]` attribute here before `param` and parameters are attributed with `[Parameter()]` and optional properties. See *eng/common/TestResources/New-TestResources.ps1* for examples, or type `help about_advanced_functions` for details.

These have a lot of benefits including pipelining input (if needed), propagating `-Verbose` and `-Debug` through to cmdlets within the script, and much more.

As written, this would work but doesn't really take advantage of PowerShell's features.

Reply:
Updated! Thanks for the suggestion :)